modules/search_api_solr_autocomplete/src/Event/SearchApiSolrAutocompleteEvents.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\search_api_solr_autocomplete\Event;
+
+/**
+ * Defines events for the Search API Solr Autocomplete module.
+ *
+ * You can also leverage any solarium event:
+ * @see https://solarium.readthedocs.io/en/stable/customizing-solarium/#plugin-system
+ * @see https://github.com/solariumphp/solarium/blob/master/src/Core/Event/Events.php
+ */
+final class SearchApiSolrAutocompleteEvents {
```

Comment:
Symfony is heading to remove event constants and use the fully qualified event class name. I think we should avoid such constants too and switch to event class names. See https://www.drupal.org/project/drupal/issues/2825358 for other reasons why we should avoid event constants.

Reply:
I commented on that issue 10 months ago ;-)

If you take a closer look you'll see that the constants' values are the class names. So you can already subscribe to the events without using the constants. Think about it as a BC layer because drupal's documentation currently describes event subscribing using such constants.